video_emotion/extract_classifier.py

```diff
@@ -0,0 +1,31 @@
+from video_emotion.facial_recognition.facial_recognition import analyze_video
+from video_emotion.emotionTagger.face_emotion_extraction import classify_face
+
+def classify_video(video_path,time_range=None):
+    faces = analyze_video(video_path,time_range)
+    angry_sum = disgust_sum = fear_sum = happy_sum = sad_sum = surprise_sum = neutral_sum = 0.0
+    number_of_faces = 0
+    for key,value in faces.items():
+        for face in value:
+            emotions = classify_face(face)
+            angry_sum += emotions[0]
+            disgust_sum += emotions[1]
+            fear_sum += emotions[2]
+            happy_sum += emotions[3]
+            sad_sum += emotions[4]
+            surprise_sum += emotions[5]
+            neutral_sum += emotions[6]
```

Comment:
Maybe add a comment showing what the ```emotions``` array contains? Not really sure though.

Reply:
I think it can be justified both ways. Problem is we use it in a lot of places, so we would have to change it everywhere. Right now we just assume that we know what the emotions array contains. It can always be check in get labels. I do agree it's a bit iffy though, maybe we could just use getlabels everywhere?